### ISSUE TITLE
Training issue - model learning nothing!

### ISSUE THREAD

**Issue by ghost** [RenMin1991/Dyamic_Graph_Representation#4]
Hi, nice work.
I tried training the network for 9000 classes of iris images normalised to 512*64 using rubber sheet algo, with total number of 25k images. I commented the resize transform. I am doing single scale first!!

The t_loss started with a value of 5 and after 5k/10k iterations the loss was just reduced to 4.6

Also there was no significant difference in matching scores and non matching scores even on training set.

Can you please help me? Let me know if you need any additional information.

**Comment by WDRBB** [comment]
I encountered the same problem, I used the multi-scale network for training, the trained model is effective for testing, but with the single-scale model for training, the trained model has almost no effect

**Comment by yangh-zzf-itcast** [comment]
> I encountered the same problem, I used the multi-scale network for training, the trained model is effective for testing, but with the single-scale model for training, the trained model has almost no effect

 Hello，I encountered the same problem.Do you have any solutions

**Comment by yangh-zzf-itcast** [comment]
> Hi, nice work. I tried training the network for 9000 classes of iris images normalised to 512*64 using rubber sheet algo, with total number of 25k images. I commented the resize transform. I am doing single scale first!!
> 
> The t_loss started with a value of 5 and after 5k/10k iterations the loss was just reduced to 4.6
> 
> Also there was no significant difference in matching scores and non matching scores even on training set.
> 
> Can you please help me? Let me know if you need any additional information.

Hello，I encountered the same problem.Do you have any solutions
